frontend_lib/src/helper.js

```diff
@@ -449,13 +449,12 @@ export const parserStringToList = (string, separatorList = [',', ';', '\n']) =>
   return parsedString.split(',').filter(notEmptyString => notEmptyString !== '')
 }
 
-// INFO - GB - 2019-07-31 - This function check if the email has three parts arranged like somethig@something.somethig
+// INFO - GB - 2019-07-31 - This function check if the email has two parts arranged like somethig@something
 export const checkEmailValidity = email => {
   const parts = email.split('@')
```

Comment:
```suggestion
  const parts = email.split('@')
```
To be consistent with the backend behavior, i suggest you to check if something like "value.rsplit("@", 1)" is possible.
The rfc say the @ to check is the last one as far i understood, and splitting will obtains weird result if you have more than 2 elements.

Reply:
I confirm that the domain is after the last @ in an email.

Reply:
Updated